README.md

```diff
@@ -1,9 +1,24 @@
 
+> At Timeplus, component is named after the [particles](https://en.wikipedia.org/wiki/List_of_particles), [chameleon](https://en.wikipedia.org/wiki/Chameleon_particle) is a hypothetical scalar particle that couples to matter more weakly than gravity, postulated as a dark energy candidate.
+
 ![Timeplus Chaneleon](chameleon.png)
+
 # Timeplus Chameleon
 
 Make stream data test easy and efficient!
 
+## What is Chameleon
+
+Chameleon is Timeplus's open source repository for stream data test tools, we are openning these code to make stream data related test easy and efficient.
```

Comment:
2 sentences. Full stop, then "We .."
Also should not be "are opening", this is already done